integrated_channels/blackboard/admin/__init__.py

```diff
@@ -38,3 +39,19 @@ def enterprise_customer_name(self, obj):
                 being rendered with this admin form.
         """
         return obj.enterprise_customer.name
+
+    def oauth_authorization_url(self, obj):
```

Comment:
this is not strictly speaking needed, but how hard would it be to make this a hyperlink in the django admin world? I do not know either, figure it's not worth it unless easy to do

Reply:
sure!